test/Test008-Resource-Service.Tests.ps1

```diff
@@ -83,10 +83,18 @@ Describe -Name 'Resource Element Tests' -Fixture {
         $RemovedResourceElement | Should Be $Null
 
     }
+    
```

Comment:
This test needs to be moved above the previous test 'Remove named Resource Element', so that it takes place before the Resource Element is deleted.